INSTALL.rst

```diff
@@ -34,7 +113,8 @@ We build assets using `npm`. Make sure you have installed it system wide.
 
 .. code-block:: console
 
-    (inspirehep)$ npm update && npm install -g node-sass clean-css requirejs uglify-js
+    (inspirehep)$ sudo npm update
+    (inspirehep)$ sudo npm install -g node-sass clean-css requirejs uglify-js
```

Comment:
FYI `npm install -g` does not require `sudo` using the following configuration: https://docs.npmjs.com/getting-started/fixing-npm-permissions